CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+
+Once you’ve finished translating OSRM Text Instructions into a new language in Transifex, open an issue in this repository asking to pull in your localization. Or do it yourself and open a pull request with the results:
+
+1. _(First time only.)_ Create a plain text file named transifex.auth in the root folder of your clone of this repository. (This file is ignored by Git, so you don’t need to worry about accidentally committing it.) Fill in your [Transifex credentials](https://docs.transifex.com/api/introduction#authentication):
+   ```json
+   {
+       "user": "username",
+       "pass": "password"
+   }
+   ```
+1. Create an empty translation file by running `echo "{}" > languages/translations/xyz.json`, where _xyz_ is a language code, such as `en`, `fil`, `pt-BR`, or `zh-Hant`.
+1. In languages.js, require the new file and add an entry for _xyz_ to the `instructions` variable.
+1. Pull the translations from Transifex by running `npm run transifex`.
+1. _(Optional.)_ If your language has special casing rules or you need to systematically change what’s imported from Transifex, add an override file to languages/overrides/_xyz_.json.
+1. _(Optional.)_ [Add an abbreviation file](languages/abbreviations/README.md) to languages/abbreviations/_xyz_.json that determines how words in road names and place names can be abbreviated when instructions are displayed visually.
+1. _(Optional.)_ [Add a grammar file](languages/grammar/README.md) to languages/grammar/_xyz_.json that specifies the declension rules to apply to road and place names in a sentence.
+1. Generate test fixtures for the language by running `UPDATE=1 npm test`. Run `git diff` to inspect the changes; make sure your translations make sense in the context of a complete sentence. Once you’re satisfied with the results, commit and push your changes.
+
+### Related projects
+
+Please consider also translating the following related projects, which work together with OSRM Text Instructions to give the user a complete turn-by-turn navigation experience:
+
+* [Mapbox Navigation SDK for Android](https://www.transifex.com/mapbox/mapbox-navigation-sdk-for-android/), which adds turn-by-turn navigation functionality to an Android application, including voice instructions powered by this library ([instructions](https://github.com/mapbox/mapbox-navigation-android/#translations))
+* [Mapbox Navigation SDK for iOS](https://www.transifex.com/mapbox/mapbox-navigation-ios/), the corresponding library for iOS applications ([instructions](https://github.com/mapbox/mapbox-navigation-ios/blob/master/CONTRIBUTING.md#adding-or-updating-a-localization))
+* [Mapbox Maps SDK](https://www.transifex.com/mapbox/mapbox-gl-native/), which is responsible for the map view and minor UI elements such as the compass ([instructions](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/ios/DEVELOPING.md#adding-a-localization))
+
+## Releasing a new version
```

Comment:
Did you mean to keep this section in the readme?

Reply:
It seemed a little too inside-baseball for developers only interested in using the library. If it’s a stretch to call releasing a form of “contributing”, we could relegate it to a separate RELEASING.md document or a wiki page, perhaps.